test/api_spec.coffee

```diff
@@ -732,8 +732,8 @@ describe "api", ->
       expect(sub_1.folders[1].path).to.eql('test_folder1/test_subfolder2')
       cloudinary.v2.api.sub_folders('test_folder_not_exists')
     ).then((result)->
-      console.log('error test_folder_not_exists should not pass to "then" handler but "catch"')
```

Comment:
Why remove the message?

Reply:
Seems like a line was removed during the merge but we don't know why (the developer doesn't remember...)